WTWSMS/common/nicknames/VIET_immersion_nicknames.txt

```diff
@@ -158,7 +164,12 @@ nick_the_realized = {
 
 nick_the_deluded = {
 	allow = {
-		religion = buddhist
+		OR = {
+			religion = buddhist
+			religion = priscillianism
+			religion = donatist
+			religion = monatism
```

Comment:
It doesn't. As you can see immediately above the additions, notice that you have to be a VEGETARIAN to get it. Those religions can become vegetarian.